labels/labels.go

```diff
@@ -17,6 +17,17 @@ const (
 	// autoOut is the label used for loop out swaps that are automatically
 	// dispatched.
 	autoOut = "autoloop-out"
+
+	// loopOutSweepSuccess is the label used for loop out swaps to sweep
+	// the HTLC in the success case.
+	loopOutSweepSuccess = "loop-out-sweep-success"
```

Comment:
I'm currently working on custom categories for faraday reports (lightninglabs/faraday/pull/85) which allow you to create a new category in your statement by matching tx labels to a regex. Perhaps add an explicit prefix var (`loopd`) and then make all these labels build on that so it's clear that we need the prefix? Nice to have a consistent format across our services building on lnd.